nemo/collections/asr/metrics.py

```diff
@@ -97,3 +98,26 @@ def classification_accuracy(
             results.append(correct_k)
 
     return results
+
+
+def classification_confusion_matrix(logits: torch.Tensor, targets: torch.Tensor) -> torch.Tensor:
+    """
+    Computes the confusion matric provided with 
```

Comment:
Nitpick: minor typo here. "matric" -> "matrix"

Reply:
Good catch! Thank you!